### DISCUSSION TITLE
Allow OpenAPI blocks inside tabs

### DISCUSSION THREAD

**Discussion by terra-alex** [GitbookIO/community#690] [Feature Requests]
**Issue:** Inside a tabs object, we want to be able to insert an OpenAPI block, but cannot at the moment.

**Reasoning:** we have 3 different flows for one given action, which are best represented by using a tabs object that allows the user to select the type of flow they'll use. Each one might have its own API call which we want to display. We're currently doing it on 3 different pages, but would prefer to have it in the same one with tabs.

**Consideration:** The width may not be sufficient for properly displaying the OpenAPI block

**Potential solution:** Allow "full width" option for tab blocks

**Comment by carina-myrand** [comment]
I was just about to request this feature too. I would also like to be able to insert OpenAPI blocks into tabs
